src/index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length--28">
+      <div class="calendar__cell calendar__cell--1"></div>
```

Comment:
```suggestion
      <div class="calendar__cell"></div>
```
Calendar elements shouldn't have any additional modifiers

Reply:
Done.